app/routes/_extras.brand.tsx

```diff
@@ -0,0 +1,345 @@
+import type { MetaFunction } from "@remix-run/node";
+import type { FC } from "react";
+
+export const meta: MetaFunction = () => {
+  return [
+    {
+      title: "Remix Assets and Branding Guidelines",
+    },
+  ];
+};
+
+export default function Brand() {
+  return (
+    <div className="prose container flex max-w-full flex-col gap-8 text-base sm:text-lg lg:max-w-4xl">
+      <h1 className="text-2xl font-extrabold dark:text-gray-200 md:text-5xl">
+        Remix Brand
+      </h1>
+      <p>
+        These assets are provided for use in situations like articles and video
+        tutorials.
+      </p>
+      <AssetHeader>Trademark Usage Agreement</AssetHeader>
+      <p>The Remix name and logos are trademarks of Shopify Inc.</p>
+      <p>
+        You may not use the Remix name or logos in any way that could mistakenly
+        imply any official connection with or endorsement of Shopify Inc. Any
+        use of the Remix name or logos in a manner that could cause customer
+        confusion is not permitted.
+      </p>
+      <p>
+        Additionally, you may not use our trademarks for t-shirts, stickers, or
+        other merchandise without explicit written consent.
+      </p>
+      <AssetHeader>Logo</AssetHeader>
+      <p>
+        Please use the logo with appropriate background. On dark backgrounds use
+        the light or glowing logo, and on light backgrounds use the dark logo. You may also{" "}
+        <a
+          content="Google drive containing all image files for Remix Logo\'s"
+          href="https://drive.google.com/drive/u/0/folders/1pbHnJqg8Y1ATs0Oi8gARH7wccJGv4I2c"
+          className="text-blue-brand hover:underline"
+        >
+          {" "}
+          download all image files for Remix Logo's in bulk through Google
+          Drive.
+        </a>
```

Comment:
This was my oversight -- let's make this open in a new tab since it goes outside of remix.run

```suggestion
        <a
          content="Google drive containing all image files for Remix Logo\'s"
          href="https://drive.google.com/drive/u/0/folders/1pbHnJqg8Y1ATs0Oi8gARH7wccJGv4I2c"
          className="text-blue-brand hover:underline"
          target="_blank"
          rel="noopener noreferrer"
        >
          download all image files for Remix Logo's in bulk through Google
          Drive.
        </a>
```